openfisca/openfisca-survey-manager
# Add conda Publish

#### Technical changes

- Add publication to conda:
  - Add GitHub Action CI
  - Publish to conda
  - Test that package work in Windows
  - Add documentation on how to use with Conda in README.md
  - Add documentation on how to publish to Conda in .conda/README.md

- [x] @sandcha : Could you check that $PYPI_USERNAME and $PYPI_PASSWORD are configured for the project ?
